src/writer/serve.py

```
@@ -381,77 +397,189 @@ def serialize_result(data):
+            except Exception as e:
+                import traceback
+                error_chunk = {
+                    "status": "error",
+                    "error": str(e),
+                    "traceback": traceback.format_exc(),
+                    "finished_at": int(time.time())
+                }
+                yield f"data: {json.dumps(error_chunk)}\n\n"
+            finally:
+                # Send final end-of-stream marker
+                yield f"data: [DONE]\n\n"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)